### PR TITLE
User-story: 12091

### PR DESCRIPTION
**Pull request by ws-mps**
Add recompute to split_out_move_lines()

### CODE REVIEW

File: addons/udes_stock/models/stock_move.py [hunk 1]
```diff
@@ -195,16 +195,17 @@ def split_out_move_lines(self, move_lines):
...
+            if self.move_orig_ids:
+                (bk_move | self).update_orig_ids(self.move_orig_ids)
+
             # When not complete, splitting a move may change its state,
             # so recompute
             incomplete_moves = (self | bk_move).filtered(lambda mv: mv.state != "done")
+            incomplete_moves.recompute()
             incomplete_moves._recompute_state()
 
             move_lines.write({"state": bk_move.state})
```
Comment (nocturnalastro) [inline addons/udes_stock/models/stock_move.py:207]: Does this also need moving?

**Review by nocturnalastro — APPROVED** [review]

File: addons/udes_stock/models/stock_move.py (hunk 1 above)
Comment (ws-mps) [reply addons/udes_stock/models/stock_move.py:207]: I don't think so, we first recompute the state of the stock.move and then update its move lines.

Comment (nocturnalastro) [reply addons/udes_stock/models/stock_move.py:207]: Is the state of the moves dependent on the move lines?

Comment (ws-mps) [reply addons/udes_stock/models/stock_move.py:207]: Yes, but not on their state.

**Review by apatrick88 — APPROVED** [review]

**Review by joe-gb — APPROVED** [review]

**Pull request merged**